themes/digital.gov/src/scss/new/_topics-button.scss

```diff
@@ -0,0 +1,101 @@
+@use "uswds-core" as *;
+
+.topics-button {
+  margin-bottom: units("105");
+
+  &__link {
+    @include u-radius("sm");
+    background: white;
+    border: 1px color("primary-vivid") solid;
+    color: color("primary-vivid");
+    display: block;
+    font-family: family("sans");
+    font-size: font-size("sans", "sm");
+    font-weight: font-weight("medium");
+    overflow: auto;
+    text-decoration: none;
```

Comment:
I replaced mixins with functions except for `u-radius` and `at-media` which we need.

Reply:
thank you for doing that